src/completed.js

```diff
@@ -0,0 +1,31 @@
+import {
+  localStorageTasks, addList,
+} from './add_remove.js';
+
+let List = JSON.parse(localStorage.getItem('List')) || [];
+
+const editCheckBox = (ID, event) => {
+  List.forEach((element) => {
+    if (element.index === ID) {
+      element.completed = event;
+    }
+    localStorageTasks(List);
+  });
+};
```

Comment:
- [ ] [Optional] It would be nice to strike through the task when you toggle the checkbox. This will improve the user experience of your application. The reason why I say this is because when I click the checkbox and reload the page, I only get to see that the status changed by looking at the `localStorage` in the browser. Kindly implement this feature. 👍